web/models/registration.ex

```diff
@@ -0,0 +1,16 @@
+defmodule ElixirBlog.Registration do
+  import Ecto.Changeset, only: [put_change: 3]
+
+  alias ElixirBlog.Repo
+
+  def create(changeset) do
```

Comment:
Not necessary, but I like to pass the Repo in so we don't couple to only one. You can default it for the best of both worlds, i.e.:

```elixir
def create(changeset, repo \\ Repo) do
  changeset
  |> put_change(:crypted_password, hashed_password(changeset.params["password"]))
  |> repo.insert()
end
```